Essentials/src/net/ess3/api/events/UserKitUseEvent.java

```diff
@@ -0,0 +1,55 @@
+package net.ess3.api.events;
+
+import org.bukkit.Bukkit;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+import com.earth2me.essentials.Kit;
+
+import net.ess3.api.IUser;
+
+/**
+ * Called when the player is given a kit
+ */
+public class UserKitUseEvent extends Event implements Cancellable {
```

Comment:
I'd prefer this was called `KitClaimEvent`, which is more descriptive about when the event is called.